content/panorama/styles/custom_game/hero_selection.css

```diff
@@ -105,13 +105,23 @@ Label {
   height: 100%;
 }
 
-#TeamRadiant {
-  flow-children: right;
+#TeamRadiant, #TeamDire {
```

Comment:
In some patch, Valve broke the usage of commas and sometimes they don't work well with their resource compiler, so if you ever get some weird unexplainable error related to hero_selection.css, be sure it's this ...

Reply:
lol dem fuks